demonstrations/ahs_aquila.py

```diff
@@ -0,0 +1,803 @@
+to Rydberg state transition. The driven Hamiltonian of the system is:
+
+.. math::  \hat{H}_{drive} = \sum_{k=1}^N \frac{\Omega(t)}{2} (e^{i \phi(t)}\ket{g_k}\bra{r_k} - e^{-i \phi(t)} \ket{r_k}\bra{g_k}) - \Delta(t) \hat{n}_k
+
+where :math:`\ket{r}` and :math:`\ket{g}` are the Rydberg and ground states respectively.
+
+Now that we know a bit about the system we will be manipulating, let us look at how to connect to a
+real device.
+
+
+
+Getting started with Amazon Braket
+----------------------------------
+
+For this demo, we will integrate PennyLane with Amazon Braket to perform measurements on Rydberg
+atom based hardware provided by QuEra.
+
+In PennyLane, Amazon Braket is accessed through the PennyLane-Braket plugin. The installation
+instructions for the plugin can be found `here <https://amazon-braket-pennylane-plugin-python.readthedocs.io/en/latest/installation.html>`__.
+
+The remote hardware devices available on Amazon Braket can be found
+`here <https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html>`__, along with
+information about each system, including which paradigm (gate-based, continuous variable or
+analog Hamiltonian simulation) it operates under. Each device has a unique identifier known as an
+ARN. In PennyLane, AHS-based Braket devices are accessed through a PennyLane device named
+``braket.aws.ahs``, along with specification of the corresponding ARN.
+
+.. note::
+
+    To access remote services on Amazon Braket, you must first
+    `create an account on AWS <https://aws.amazon.com/braket/getting-started/>`__ and also follow the
+    `setup instructions <https://github.com/aws/amazon-braket-sdk-python>`__ for accessing Braket from Python.
+
+Connecting to Aquila
+~~~~~~~~~~~~~~~~~~~~
+
+Once you are set up with a Braket account, you can access the Aquila device. It is available online
+in particular time windows, which can be found `here <https://www.quera.com/aquila>`__ , though you
+can upload tasks to the queue at any time. Note that depending on queue lengths, there can be some
+wait-time to receive results even during the availability window of the device.
+
+A simulated version on the Aquila hardware, ``braket.local.ahs``, is also available, and is an
+excellent resource for testing out programs before committing to a particular hardware task. It
+is important to be aware that some things that succeed in simulation will not be able to be sent
+to hardware due to physical constraints of the measurement and control setup. It is important to
+be aware of the hardware specifications and capabilities when planning your pulse program. These
+capabilities are accessible at any time from the hardware device; we will demonstrate in more
+detail where to find these specifications and where they are relevant as we go through this demo.
+
+.. note::
+
+    Some cells of this notebook will only run when hardware is online. If you want to run it at
```

Comment:
```suggestion
    Those cells of this notebook that contain the real device ``aquila`` will only run when hardware is online. If you want to run it at
```
To make it more precise what won't work without hardware access, and in particular, what does work without it :)